test/common.bash

```diff
@@ -30,6 +30,11 @@ env_require () {
     fi
 }
 
+fatal () {
+    printf '%s\n\n' "$1" >&2
+    exit 1
+}
+
```

Comment:
I like abstracting this into a function.

I'd like for this PR to change the other error exits to use it too.